devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java

```diff
@@ -76,9 +77,18 @@ static QuarkusPlatformDescriptor resolvePlatformDescriptor(final String bomGroup
         String artifactId = StringUtils.defaultIfBlank(bomArtifactId, null);
         String version = StringUtils.defaultIfBlank(bomVersion, null);
 
-        if (CreateUtils.QUARKUS_CORE_BOM_ARTIFACT_ID.equals(artifactId)
-                && version == null) {
-            version = resolvePluginInfo(CreateUtils.class).getVersion();
+        if (version == null) {
+            if (CreateUtils.QUARKUS_CORE_BOM_ARTIFACT_ID.equals(artifactId)) {
+                version = resolvePluginInfo(CreateUtils.class).getVersion();
+            } else if ((groupId == null && artifactId == null && version == null)
+                    || ("quarkus-universe-bom".equals(artifactId) && version == null)) {
```

Comment:
Looks good. Now, given that the outer `if` is checking the version for null, we can be sure that the version will be null inside its block. So you don't have to be checking it again in the nested `if`s.

Reply:
Oh yes I am sorry IDK how I skipped those